src/Application/Repositories/IRequisitionRepository.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Domain.Entities.Aggregates.RequisitionAggregate;
+
+namespace Application.Repositories
+{
+    public interface IRequisitionRepository
+    {
+        Task<Requisition> AddAsync(Requisition requisition);
+        Requisition Update(Requisition requisition);
```

Comment:
Make it async.

Reply:
I made it synchronus since EfCore doesn't have UpdateAsync and the method would only run synchronously plus there was a warning to that effect in the IDE. Should I still make it Async regardless sir?

Reply:
yes

Reply:
Done